src/plugins/calculationTry1/countCellsData.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default function countCellsData(prop: JSX.Element) {
+  function handleKeyPress(e: React.KeyboardEvent<HTMLElement>) {
+    let countString = (e.target as Element).innerHTML;
+    if (e.key === "Enter" && countString[0] === "=" && (e.target as Element).classList.contains("data-input")) {
```

Comment:
countString[0].equals("=")

Reply:
e.key.equals('Enter')

Reply:
`e.key.equals('Enter')` ts said: "Property 'equals' does not exist on type 'string'."
It'd be better to write `e.key === "Enter"` or  to solve problem with ts?